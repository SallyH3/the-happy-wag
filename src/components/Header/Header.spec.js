import React from 'react';
import { Header } from './index';
import { shallow } from 'enzyme';
import { createSerializer } from 'enzyme-to-json';

expect.addSnapshotSerializer(createSerializer({mode: 'deep'}));

describe('Header', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(
      <Header />
    )
  });

  it('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  })
})