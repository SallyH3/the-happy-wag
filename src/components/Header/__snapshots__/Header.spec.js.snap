// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should match snapshot 1`] = `
<section
  className='header-wrapper'
>
  <article
    className='header'
  >
    <h1
      className='main-title'
    >
      The Happy Wag
    </h1>
  </article>
  <article
    className='subtitle'
  >
    <p
      className='sub-text'
    >
      Browse through the furry companion cards and click on an animal for contact information to schedule a meet 'n' greet!
    </p>
  </article>
</section>
`;
